_posts/2023-10-04-new-library-updates.md

```diff
@@ -13,7 +13,7 @@ Along with 2.1, we are also releasing a series of beta updates to the PyTorch do
 | Latest Stable Library Versions  |([Full List](https://pytorch.org/docs/stable/index.html))*                  |                             |
 |--------------------------------------------|------------------|-----------------------------|
 | TorchArrow 0.1.0                           | TorchRec 0.4.0   | TorchVision 0.16            |
```

Comment:
TorchRec should read 0.5.0. @agunapal  can you make this change as well in this PR?

Reply:
Done @colin2328